vars/githubscm.groovy

```diff
@@ -262,6 +262,30 @@ def removeRemoteTag(String remote, String tagName, String credentialsId = 'kie-c
     println "[INFO] Deleted remote tag ${tagName}."
 }
 
+/*
+* Checks if a release already exists, removes it and creates a new 
+*/
+
+void createRelease(String tagName, String buildBranch, String credentialsId = 'kie-ci') {
+    def String description = "release ${tagName}"
```

Comment:
```suggestion
    String description = "release ${tagName}"
```

Reply:
Also can we make the description a parameter of the method, with default to `Release ${tagName}`